index.js

```diff
@@ -18,8 +18,10 @@ var country = function(country) {
   });
 };
 var number = function(number) {
+  number = ('000' + number).slice(-3);
```

Comment:
I could've used `String(number).padStart(3, '0')`, but that requires Node 8. I don't know which version of Node you're intending to support, not to mention browsers, but given that all the JavaScript I see here is fairly legacy, I went with the safe option.